### PR TITLE
Removing login_hint from signout

### PR DESCRIPTION
**Pull request by TiagoBrenck**
## Purpose
Removed **login_hint** and **domain_hint** from `OnRedirectToIdentityProviderForSignOut` since it is not supported on OIDC v2 anymore, due to security concerns. The change is already done on EVO side, so this is just a clean-up to avoid misinformation. 

## Does this introduce a breaking change?
This was supported when the samples were created, but now it is not.
```
[x] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Documentation content changes
[ ] Other... Please describe:
```

## What to Check
Since login_hint is not supported anymore, you will notice that everytime you logout, the account selection window will be prompted.

### CODE REVIEW

File: 1-WebApp-OIDC/1-6-SignOut/README.md [hunk 1]
```diff
@@ -65,13 +65,10 @@ public static IServiceCollection AddAzureAdV2Authentication(this IServiceCollect
         ...
         options.Authority = options.Authority + "/v2.0/";
         ...
-        // Attempt to avoid displaying the select account dialog when signing out
+        
         options.Events.OnRedirectToIdentityProviderForSignOut = async context =>
         {
```
Comment (kalyankrishna1) [inline 1-WebApp-OIDC/1-6-SignOut/README.md]: I'd suggest removing the doc bits completely as well. But I'll let others weigh in as well.

File: 1-WebApp-OIDC/1-6-SignOut/README.md [hunk 2]
```diff
@@ -65,13 +65,10 @@ public static IServiceCollection AddAzureAdV2Authentication(this IServiceCollect
         ...
         options.Authority = options.Authority + "/v2.0/";
         ...
-        // Attempt to avoid displaying the select account dialog when signing out
+        
         options.Events.OnRedirectToIdentityProviderForSignOut = async context =>
```
Comment (jmprieur) [inline 1-WebApp-OIDC/1-6-SignOut/README.md]: You can probably remove the event handler altogether?

File: 1-WebApp-OIDC/1-6-SignOut/README.md (hunk 1 above)
Comment (jmprieur) [reply 1-WebApp-OIDC/1-6-SignOut/README.md]: snap :)

File: 1-WebApp-OIDC/1-6-SignOut/README.md (hunk 2 above)
Comment (TiagoBrenck) [reply 1-WebApp-OIDC/1-6-SignOut/README.md]: The event handler is there as an illustration for the section above titled: "Intercepting the call to the logout endpoint". I think it makes sense to keep it there with a blank implementation. Do you agree?

**Pull request merged**